models/models.js

```diff
@@ -80,10 +80,30 @@ function insertComment(article_id, username, body) {
     });
 }
 
+function updateVote(article_id, inc_votes) {
+  if (!inc_votes) {
```

Comment:
PSQL will throw an error if our inc vote value is not there, so let your error handler handles this via a code :)

Reply:
What we should deal with in this model is a 404 for the article not being found